build.gradle

```diff
@@ -200,10 +200,12 @@ subprojects {
 
         implementation 'org.slf4j:slf4j-api:1.7.30'
 
+
         // SLF4J as a facade over Log4j2 required dependencies
         implementation 'org.apache.logging.log4j:log4j-api:2.11.0'
         implementation 'org.apache.logging.log4j:log4j-core:2.11.0'
         implementation 'org.apache.logging.log4j:log4j-slf4j-impl:2.11.0'
+        implementation 'org.apache.logging.log4j:log4j-web:2.14.1'
```

Comment:
What is it for?

Reply:
I was seeing this warning https://stackoverflow.com/questions/39420059/log4j2-no-log4j-web-module-available and it went away as soon as I added the jar